cypress/integration/pages/frontPage/tests.js

```diff
@@ -1,9 +1,34 @@
+import * as moment from 'moment-timezone';
+import config from '../../../support/config/services';
+import appConfig from '../../../../src/app/lib/config/services';
 // For testing important features that differ between services, e.g. Timestamps.
 // We recommend using inline conditional logic to limit tests to services which differ.
 const serviceHasIndexAlsos = service => service === 'thai';
 
+const serviceHasTimestamp = service => service === 'thai';
+
 export const testsThatAlwaysRun = ({ service, pageType }) => {
-  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+  describe(`Running testsToAlwaysRun for ${service} ${pageType}`, () => {
+    if (serviceHasTimestamp(service)) {
+      it('should render a formatted timestamp in the top story', () => {
+        cy.request(`${config[service].pageTypes.frontPage.path}.json`).then(
+          ({ body }) => {
+            const { language } = body.metadata;
+            const { timestamp } = body.content.groups[0].items[0];
+            const formattedTimestamp = moment
+              .tz(timestamp, `${appConfig[service].timezone}`)
+              .locale(language)
+              .format('D MMMM YYYY');
+            cy.get('section li')
```

Comment:
This doesn't get the `topStory`. as top story is not in a list.
```suggestion
            cy.get('section')
```

Reply:
Fixed :)